### PR TITLE
docs: Update README.md

### LINKED ISSUE #591: README.md should be updated
## Description
As a developer,
In windows, I need to activate the virtual environment using '.\venv\Scripts\activate', but the README.md shows './venv/Scripts/activate'.
so that I can activate the virtual environment correctly.

### Update [Required]
- [ ] Update README.md file.

## Estimation
1 hour

### PR DESCRIPTION
**Pull request by khushishikhu**
Description
Changed in code so that virtual environment can activate correctly.

Fixes #591 

Type of Change:

- Code
- Documentation

Checklist:

- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by khushishikhu** [comment]
@SanketDG  please review my changes.

**Comment by SanketDG** [comment]
Make sure to follow the [commit message guidelines](https://github.com/anitab-org/portal/wiki/Commit-Message-Style-Guide)

**Comment by khushishikhu** [comment]
> Make sure to follow the [commit message guidelines](https://github.com/anitab-org/portal/wiki/Commit-Message-Style-Guide)

yeah sure. Thank you.

**Comment by SanketDG** [comment]
You will need to change the commit message, not the PR title.

**Comment by khushishikhu** [comment]
@SanketDG  i have changed the commit message style. can you please review it . if it's okay or not now.

File: README.md [hunk 1]
```diff
@@ -106,7 +106,7 @@ Setup for developers (Windows)
  
      ```bash
      $ virtualenv venv
-     $ ./venv/Scripts/activate
+     $ .\venv\Scripts\activate
```
Comment (SanketDG) [inline README.md:109]: Do you not need a `.bat` for the extension?

Comment (khushishikhu) [reply README.md:109]: i think in CMD `.\venv\Scripts\activate` and `.\venv\Scripts\activate.bat` both can work. Problem arises when we are working on windows powershell.

Comment (khushishikhu) [reply README.md:109]: as in windows powershell we can not use `.\venv\Scripts\activate.bat`

**Review by SanketDG — APPROVED** [review]
Looks good to me!

Should be merged with a second approval.

**Comment by khushishikhu** [comment]
@SanketDG  Thank you for your approval!!

**Comment by khushishikhu** [comment]
@saideepesh000 can you please review my changes.

**Comment by saideepesh000** [comment]
> 
> 
> @saideepesh000 can you please review my changes.

@khushishikhu, thanks for mentioning me, actually I am not the mentor or maintainer of this project, thanks.

**Comment by SanketDG** [comment]
@saideepesh000 

Anyone can review a PR, you can too, it would be of great help!

Whether it's merged or not is upon the maintainers.

**Comment by khushishikhu** [comment]
@SanketDG  can you please recommend or request someone to review my PR.

**Comment by khushishikhu** [comment]
@saideepesh000  okay, thank you.

**Comment by khushishikhu** [comment]
@sammy1997  can you please review my changes.

**Review by sakshi1499 — APPROVED** [review]
The commit message check is still failing, otherwise looks good. Whoever merges it, please see that it is taken care of.

**Comment by khushishikhu** [comment]
@sakshi1499  Thank you for your approval !!

**Pull request merged**